src/gwc/src/test/java/org/geoserver/gwc/layer/DefaultTileLayerCatalogTest.java

```diff
@@ -93,6 +94,44 @@ public void tearDown() throws Exception {
             info.setName("name1");
             info.getMimeFormats().add("image/png");
             info.getMimeFormats().add("image/jpeg");
+
+            assertNull(catalog.save(info));
+
+            original = catalog.getLayerById("id1");
+            assertEquals(info.getMimeFormats(), original.getMimeFormats());
+        }
```

Comment:
These scope blocks seem odd, what is the reason for them?

Reply:
Not sure? Just copied and pasted from the previous test to be honest.

Reply:
Fair enough, seems fine to merge.